base/loading.jl

```diff
@@ -173,18 +173,23 @@ function _require_from_serialized(node::Int, mod::Symbol, path_to_try::String, t
         restored = _include_from_serialized(content)
         isa(restored, Exception) && return restored
         others = filter(x -> x != myid(), procs())
-        refs = Any[
-            (p, @spawnat(p,
-                let m = try
-                            _include_from_serialized(content)
-                        catch ex
-                            isa(ex, Exception) ? ex : ErrorException(string(ex))
-                        end
-                    isa(m, Exception) ? m : nothing
-                end))
-            for p in others ]
-        for (id, ref) in refs
-            m = fetch(ref)
+
+        results = Any[]
```

Comment:
`sizehint!(Vector{Tuple{Int,Any}}(), nprocs())`?

Looks like `others` is not required, should be ok to just skip `myid()` before `@async` block. One allocation less :)